src/api/selection.js

```diff
@@ -223,13 +138,9 @@ function (elementHelper) {
       var newRange = document.createRange();
 
       newRange.setStartBefore(markers[0]);
-      if (markers.length >= 2) {
-        newRange.setEndAfter(markers[1]);
-      } else {
-        // We always reset the end marker because otherwise it will just
-        // use the current range’s end marker.
-        newRange.setEndAfter(markers[0]);
-      }
+      // We always reset the end marker because otherwise it will just
+      // use the current range’s end marker.
+      newRange.setEndAfter(markers[1] || markers[0]);
```

Comment:
I think is quite confusing, could you inline a ternary operator here to have something a little clearer?